src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -3868,24 +3868,6 @@ public async Task Finalizer()
             await AssertFormatAsync(expected, code);
         }
 
-        [Fact]
-        [WorkItem(538743, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/538743")]
-        [Trait(Traits.Feature, Traits.Features.Formatting)]
-        public async Task BugFix4442()
-        {
-            var code = @"class Program
-{
-    static void Main(string[] args)
-    {
-        string str = ""ab,die|wo"";
-        string[] a = str.Split(new char[] { ',', '|' })
-            ;
-    }
-}";
-
-            await AssertFormatAsync(code, code);
-        }
```

Comment:
I'm not sure what is this test supposed to be testing, but I assume it's that semicolon on a new  line won't be reformatted, which would mean that deleting it is the right approach.